plainly-plugin/src/ui/components/form/UploadForm.tsx

```diff
@@ -0,0 +1,170 @@
+import FormData from 'form-data';
```

Comment:
again getting this:

![image](https://github.com/user-attachments/assets/d5caca54-6211-4cff-814b-e4f4d823aa32)